packages/gestalt/src/Flex.tsx

```diff
@@ -5,7 +5,27 @@ import FlexItem from './FlexItem';
 import wrapWithComponent from './utils/wrapWithComponent';
 
 type Dimension = number | string;
-type Gap = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16;
+type Gap =
+  | 0
+  | 0.25
+  | 0.5
+  | 1
+  | 1.5
+  | 2
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10
+  | 11
+  | 12
+  | 13
+  | 14
+  | 15
+  | 16;
```

Comment:
Just a thought - If the change is here - and we're not moving to the 100 scale in the actual TS props.. do we really need to migrate all of the internal classes to have a 100 in the naming? 

Isn't it all minified in the build - this PR seems to touch a lot of files, but I'm not sure if there is a clear benefit since the classes are auto-generated

Reply:
It's one step closer to tokens and o.25 becomes 250 rather than 25 or 025, and it follows the tokens underneath.

Reply:
i cannot have clannsmaes that are `margin0.25`

Reply:
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/b9785f34-e049-4241-996d-ea1157752820)


Next will be moving from 1 to 100 in the API. And all we be aligned.